Remotion/Validation/Core/Validators/EqualValidator.cs

```diff
@@ -15,38 +26,39 @@ public class EqualValidator : PropertyValidator, IValueComparisonValidator
     public object ValueToCompare { get; }
```

Comment:
IComparable

Reply:
Leaving this as it is, as `EqualValidator` gets an `object` through the ctor.